RedHat-UX/red-hat-design-system
# docs(rh-code-block): add missing image and fix summary

Closes  #1162

## What I did

1. Added rh-code-block's missing screenshot image
2. Capitalized the first word in the summary


## Testing Instructions

1. Check the DP and verify that the code block image is listed with the new summary
